front_end/content.py

```diff
@@ -140,13 +140,44 @@ def get_role(self, user_id):
         else:
             return "student"
 
+    def get_users_from_role(self, role, course_id):
+        users = []
+        if role == "administrator":
+            sql = 'SELECT user_id FROM permissions WHERE role=?'
```

Comment:
I tried this but got an error: "sqlite3.OperationalError: incomplete input"

Reply:
I don't know offhand what would cause the error (probably something off with the syntax), but dig into it some more and see if you can get it to work. You may need to test it in DB Browser for SQLite.

Reply:
I got it!